Update documentation for web servers and webots.cloud (#4797)
cyberbotics/webots: docs/guide/advanced-configuration.md

## Advanced Configuration

### Docker

If the `docker` configuration option is set to `true`, Docker will be used to start Webots, otherwise Webots will be started directly on the metal of the server.
If the simulation project contains a `Dockerfile` file at the root level, this file will be used to start the Webots instance inside the corresponding Docker container.
Otherwise, a `Dockerfile` will be created based on the standard Docker image of Webots.
The version of Webots for the Docker image is automatically computed from the header line of the simulation world file.
For example if the world file starts with the following line:

```
#VRML_SIM R2022a utf8
```

The simulation server will create a `Dockerfile` starting with:
```
FROM docker image cyberbotics/webots:R2022a-ubuntu20.04
```

> Running Webots inside a Docker container is a very little overhead, but guarantees that the simulation server remain secure, regardless of the running simulations.

Reviewer: ```suggestion
Running Webots inside a Docker container is a very little overhead, but guarantees that the simulation server remains secure, regardless of the running simulations.
```

> If you don't want to use Docker, you should ensure that the list of `allowedRepositories` provided in the configuration file doesn't contain any malware, otherwise, you are putting your simulation server at risk.

Reviewer: ```suggestion
If you don't want to use Docker, you should ensure that the list of `allowedRepositories` provided in the configuration file doesn't contain any links to malicious repositories, otherwise, you are putting your simulation server at risk.
```

So, it can be a good idea to decrease the graphical settings to increase the number of instances of Webots that could run in parallel.
By changing the graphical settings (that are present in the `Preferences/OpenGL` tab of Webots) from maximum to minimum, it is possible to multiply the number of parallel instance by 3 or 4 if your GPU is the bottleneck.

To do that, you have to modify the configuration file of Webots (in `/$Home/.config/Cyberbotics/Webots-R202??.conf`), then create a new docker image. You can use the following Dockerfile:
```
FROM my_name/my_webots_repo:previous_webots_tag
COPY /path_to_the_configuation_file/Webots-R202??.conf /root/.config/Cyberbotics/Webots-R202??.conf
```

#### Put the Assets in the Docker
Webots (in the docker) will have to load the world first. To reduce this loading time, it is possible to put the assets directly in the docker such that Webots will not need to download them from the web.

You can achieve that with the following Dockerfile:
```
FROM my_name/my_webots_repo:previous_webots_tag
COPY path_to_your_assets_folder/assets /root/.cache/Cyberbotics/Webots/assets
```

> 

Reviewer: ```suggestion

To work, your assets folder must correspond to the version of Webots that is in the docker.

```
